pymidi/protocol.py

```diff
@@ -67,17 +67,17 @@ def sendto(self, message, addr):
 
     def handle_message(self, data, addr):
         if self.logger.isEnabledFor(logging.DEBUG):
-            self.logger.debug('rx: {}'.format(data.encode('hex')))
+            self.logger.debug('rx: {}'.format(data.hex()))
 
         try:
             if data[0:2] == APPLEMIDI_PREAMBLE:
                 command = data[2:4]
-                self.logger.debug('Command: {}'.format(command))
+                self.logger.debug('Command: {}'.format(command.hex()))
```

Comment:
When I start a server up and try to connect to it from my mac, I get an error at this line:

> AttributeError: 'newbytes' object has no attribute 'hex'

It seems like this is a problem with python2:

>$ python --version
> Python 2.7.9

so I tried running the server with python3:

> $ python3 --version
> Python 3.4.2
> $ python3 pymidi/server.py -b 0.0.0.0:5053 -b :::5051
> Traceback (most recent call last):
>   File "pymidi/server.py", line 13, in <module>
>     from pymidi.protocol import DataProtocol
> ImportError: No module named 'pymidi'

Python versions and future and package management stuff is definitely not in my wheelhouse.  My fix would be to kill .hex() and let format figure it out, but thought I see what you think.

Reply:
(I only PR'd this today, so no worries!)

Ah, bugger. Seems I've screwed up the `bytes` handling between py2/py3 - that's what I get for leaving this path untested. Let me revise this and add some test coverage, good catch, thanks!